modular_skyrat/modules/opposing_force/code/equipment/rifles.dm

```diff
@@ -5,30 +5,10 @@
 	item_type = /obj/item/gun/ballistic/automatic/m16
 	admin_note = "WARNING: This weapon is extremely powerful, firing a 3 round burst of 35 damage bullets."
 
-/datum/opposing_force_equipment/rifle/m16m
-	item_type = /obj/item/gun/ballistic/automatic/m16/modern
-	admin_note = "WARNING: This weapon is extremely powerful, firing a 3 round burst of 30 damage bullets. The modern version features a halfed spread and reduced fire delay."
-
-/datum/opposing_force_equipment/rifle/m16patriot
-	item_type = /obj/item/gun/ballistic/automatic/m16/modern/v2
-	admin_note = "WARNING: This weapon is extremely powerful, firing a 4 round burst of 30 damage bullets. The patriot version features a halfed fire delay."
-
 /datum/opposing_force_equipment/rifle/akm
 	item_type = /obj/item/gun/ballistic/automatic/akm
```

Comment:
I've made this "review" (because it was meant to be a single comment) like yesterday but thanks to phone GitHub not being that useable it not only got delayed but also became a review instead of a single comment.

Reply:
i'll get around to it in a day or two